src/main/java/carbonconfiglib/gui/impl/minecraft/MinecraftConfig.java

```diff
@@ -143,14 +143,14 @@ public void restoreDefault() {
 	public List<IConfigTarget> getPotentialFiles() {
 		LevelStorageSource storage = Minecraft.getInstance().getLevelSource();
 		List<IConfigTarget> folders = new ObjectArrayList<>();
-		for(LevelSummary sum : storage.loadLevelSummaries(storage.findLevelCandidates()).join()) {
+		/*for(LevelSummary sum : storage.loadLevelSummaries(storage.findLevelCandidates()).join()) {
```

Comment:
throw the for loop into a Try/Catch and use the function that returns a list of LevelSummaries. There is only 1 function.